data/SampleData/Hl7v2/ADT01-28.hl7

```diff
@@ -1,4 +1,4 @@
-﻿MSH|^~\&|ADT1|GOOD HEALTH HOSPITAL|GHH LAB, INC.|GOOD HEALTH HOSPITAL|198808181126|SECURITY|ADT^A01^ADT_A01|MSG00001|P|2.8||
+﻿MSH|^~\&|ADT1|GOOD HEALTH HOSPITAL|GHH LAB, INC.|GOOD HEALTH HOSPITAL|198808181126|SECURITY|ADT^A01^ADT_A01|MSG00001|P|2.5.1||
```

Comment:
Are the errors pointed out by the validator?

Reply:
Yes, I've tried with multiple validators, actually the `2.8` version number can be barely identified. What's worse is that the `2.8` version number will cause infinite loop(function invoking) of `nHapi` and finally throw a `StackOverflow` exception.